crates/orchestrator/src/tests/workers/proving/mod.rs

```diff
@@ -91,8 +97,19 @@ async fn test_proving_worker(#[case] incomplete_runs: bool) -> Result<(), Box<dy
     .await;
     config_force_init(config).await;
 
+    let y: Arc<Box<dyn Job>> = Arc::new(Box::new(job_handler));
```

Comment:
is there a way to tell mockall to call the actual function in some test cases? because we would need that for e2e

Reply:
Will have to look into it. There should be a way.